openedx/core/djangoapps/catalog/utils.py

```diff
@@ -61,6 +78,46 @@ def get_programs(user, uuid=None, type=None):  # pylint: disable=redefined-built
         return []
 
 
+def get_program_types(user=None):  # pylint: disable=redefined-builtin
+    """Retrieve all program types from the catalog service.
+
+    Returns:
+        list of dict, representing program types.
+    """
+    catalog_integration = CatalogIntegration.current()
+    if catalog_integration.enabled:
+        user = _get_service_user(user, catalog_integration.service_username)
+        if not user:
+            return []
```

Comment:
Can you add a test to cover this line? See the [diff-coverage-report](https://build.testeng.edx.org/job/edx-platform-python-unittests-pr/29609/Diff_Coverage_Report/).